README.md

```diff
@@ -8,16 +8,165 @@ Local.
 For more information, see [Getting started with Cloudant NoSQL DB](https://www.ng.bluemix.net/docs/services/Cloudant/index.html)
 
 The `loopback-connector-cloudant` module is the Cloudant connector for the Loopback framework.
+<!-- TOC -->
+
+- [loopback-connector-cloudant](#loopback-connector-cloudant)
```

Comment:
Can we add this hyperlink to https://github.com/strongloop/loopback-connector-cloudant/pull/116/files#diff-04c6e90faac2675aa89e2176d2eec7d8L1 instead?